app/scripts/controllers/pod.js

```diff
@@ -87,6 +87,58 @@ angular.module('openshiftConsole')
       }
     };
 
+    /**
+     *  Will set the newTerm's isActive/isUsed to true, while deactivating the old's
+     */
+    $scope.onTerminalSelectChange = function(newTerm) {
+      // Deactivate all terminals
+      _.each($scope.containerTerminals, function(term) {
+        term.isActive = false;
+      });
+
+      newTerm.value.isActive = true;
+      newTerm.value.isUsed = true;
+    };
+
+    $scope.tabSelect = function() {
+      $scope.terminalTabWasSelected = true;
+    };
+
+    var makeTerminals = function(containerStatuses, existingTerms) {
+      var firstRun = false;
```

Comment:
this whole block could be replaced by

```
var firstRun = !!existingTerms;
```

We always use the `!!` if we want to transform a non-boolean to a boolean.